### PR TITLE
Handle null data

### LINKED ISSUE #44: break renderables into smaller components

### PR DESCRIPTION
**Pull request by boygirl**
cc/ @coopy 

closes #44 and #46

### CODE REVIEW

File: src/components/victory-line.jsx [hunk 1]
```diff
@@ -283,6 +285,22 @@ export default class VictoryLine extends React.Component {
     return props.y;
   }
 
+  getDataSegments() {
+    const orderedData = _.sortBy(this.dataset, "x");
+    const segments = [];
+    let segmentStartIndex = 0;
+    _.each(orderedData, (datum, index) => {
+      if (datum.y === null || datum.y === undefined) {
```
Comment (coopy) [inline src/components/victory-line.jsx]: I prefer to use lodash methods: `_.isNull` & `_.isUndefined`

File: src/components/line-segment.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React, { PropTypes } from "react";
```
Comment (coopy) [inline src/components/line-segment.jsx:1]: Nice factoring-out of a sub-component!

Can I haz a small, basic test case? :]

**Comment by boygirl** [comment]
@coopy you _can_ haz test case

File: test/client/spec/components/victory-line.spec.jsx [hunk 3]
```diff
@@ -1,21 +1,126 @@
...
+        {x: 5, y: 3},
+        {x: 6, y: 4},
+        {x: 7, y: 6}
+      ];
+      renderedComponent = TestUtils.renderIntoDocument(<VictoryLine data={data}/>);
+      const path = TestUtils.scryRenderedDOMComponentsWithTag(renderedComponent, "path");
+      expect(path.length).to.equal(1);
+    });
+
+    it("renders two line segments when there are two continuous sections of data", () => {
```
Comment (coopy) [inline test/client/spec/components/victory-line.spec.jsx:50]: I don't get why this is "two continuous sections of data"?

Comment (boygirl) [inline test/client/spec/components/victory-line.spec.jsx:50]: data sorted by x... I guess it's discrete values, but we would be drawing a continuous line except for the null discontinuity.

**Comment by coopy** [comment]
:+1: LGTM after conflict resolution. Great stuff!

**Pull request merged**